src/app/core/components/sobv-page-not-found/sobv-page-not-found.component.scss

```diff
@@ -0,0 +1,30 @@
+.container {
+  min-height: 80vh;
+  display: grid;
+  place-items: center;
+
+}
+
+.not-found {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.blank-page__container {
+  position: relative;
+}
+
+.blank-page__number {
```

Comment:
We don't use BEM metodolody, use onlyObject-Oriented CSS like Bootsrap

Reply:
plus